File: payroll-api/src/main/java/com/company/payroll/mapper/StaffBankingInfoMapper.java
package com.company.payroll.mapper;

import java.util.List;

import com.company.payroll.model.StaffBankingInfo;
import org.springframework.stereotype.Repository;

@Repository
public interface StaffBankingInfoMapper {
    int deleteByPrimaryKey(Integer bId);
    int insert(StaffBankingInfo row);
    int insertSelective(StaffBankingInfo row);
    List<StaffBankingInfo> selectList();
    StaffBankingInfo selectByPrimaryKey(Integer bId);
    int updateByPrimaryKeySelective(StaffBankingInfo row);
    int updateByPrimaryKey(StaffBankingInfo row);
}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/mapper/StaffBankingInfoMapper.kt
package com.company.payroll.mapper

import com.company.payroll.model.StaffBankingInfo
import org.springframework.stereotype.Repository

@Repository
interface StaffBankingInfoMapper {
  fun deleteByPrimaryKey(bId: Int): Int

  fun insert(row: StaffBankingInfo): Int

  fun insertSelective(row: StaffBankingInfo): Int

  fun selectByPrimaryKey(bId: Int): StaffBankingInfo

  fun updateByPrimaryKeySelective(row: StaffBankingInfo): Int

  fun updateByPrimaryKey(row: StaffBankingInfo): Int
}